src/test/java/uk/ac/ebi/ampt2d/accession/serial/block/MonotonicRangeTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ *
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.accession.serial.block;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(SpringRunner.class)
+public class MonotonicRangeTest {
+
+    @Test
+    public void assertThatNoAccessions() {
+        assertTrue(MonotonicRange.convertToMonotonicRanges().isEmpty());
+    }
+
+    @Test
+    public void assertArrayOfOneRange() {
+        List<MonotonicRange> ranges = MonotonicRange.convertToMonotonicRanges(1, 2);
+        assertEquals(1, ranges.size());
+        assertEquals(1, ranges.get(0).getStart());
```

Comment:
A helper method will make those asserts more readable:

assertRangeEquals(int rangeIndex, long rangeStart, long rangeEnd);

For this method:
assertRangeEquals(0, 1, 2);

Reply:
Agree